azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+  branches:
+    include: ['*']
+  tags:
+    include: ['*']
+
+jobs:
+  - job: Test
+  pool:
+    vmImage: 'Ubuntu-18.04'
+    vmImage: 'macOS-10.13'
+    vmImage: 'vs2017-win2016'
```

Comment:
Is this line correct?

Reply:
Or we have add separate blocks for each vm.

Reply:
No idea. That's for you to figure out